app/controllers/posts_controller.rb

```diff
@@ -27,6 +27,13 @@ def create
     end
   end
 
+  def destroy
+    @post = Post.find(params[:id])
+    @post.destroy
+    redirect_to user_posts_path(@post.author)
+    authorize! :destroy, @post
+  end
+
```

Comment:
- [ ] Let's use `CanCanCan` for Post authorization in Post Controller by adding this line `load_and_authorize_resource`.